### DISCUSSION TITLE
Acronyms, Symbols...

### DISCUSSION THREAD

**Discussion by Martafct** [joaomlourenco/novathesis#403] [Q&A] [answered]
Hello. Now, I think I have everything working out okay. I downloaded the last version 30 min ago, and it worked out perfectly. 

I still have this one, about the acronyms and so on. 

I only changed the words inside gls{}, and it turned out okay, finally. But it disappears from de chapters Acronyms and so on. Do I need to do anything else, besides adding gls? Thank you so much.

**Comment by Martafct** [comment, marked as answer]
Okay, I think I got it. I needed to use it at least two times in the text. At least that worked out for me.

**Reply by joaomlourenco** [reply]
One time  in the text is enough.  But if you don't use it… it does not exist!  😉
